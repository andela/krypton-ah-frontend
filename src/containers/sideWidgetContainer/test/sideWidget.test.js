import React from 'react';
import { shallow } from 'enzyme';
import Widget from '..';

describe('button test', () => {
  it('should test for the button component', () => {
    const wrapper = shallow(<Widget />);
    expect(wrapper).toMatchSnapshot();
  });
});
