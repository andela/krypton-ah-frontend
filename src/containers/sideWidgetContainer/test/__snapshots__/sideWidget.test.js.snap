// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button test should test for the button component 1`] = `
<GridRow
  columns={1}
>
  <GridColumn>
    <header
      text="Trending"
    />
    <featuredwidget
      className="featured-widget"
      description="When bobbing for apples, an idealist endlessly reaches for the best apple, a pessimist "
      title="Apple's news at CES 2019 shows it's facing some hard truths"
    />
    <header
      text="Tags"
    />
    <List
      className="tags"
      divided={true}
      relaxed={true}
    >
      <ListItem>
        <ListContent>
          <ListHeader
            as="a"
          >
            #Politics
          </ListHeader>
        </ListContent>
      </ListItem>
      <ListItem>
        <ListContent>
          <ListHeader
            as="a"
          >
            #Technology
          </ListHeader>
        </ListContent>
      </ListItem>
      <ListItem>
        <ListContent>
          <ListHeader
            as="a"
          >
            #Family
          </ListHeader>
        </ListContent>
      </ListItem>
      <ListItem>
        <ListContent>
          <ListHeader
            as="a"
          >
            #Family
          </ListHeader>
        </ListContent>
      </ListItem>
      <ListItem>
        <ListContent>
          <ListHeader
            as="a"
          >
            #Family
          </ListHeader>
        </ListContent>
      </ListItem>
    </List>
    <GridColumn
      className="Categories"
    >
      <header
        text="Categories"
      />
      <List
        divided={true}
        relaxed={true}
      >
        <ListItem>
          <ListContent>
            <ListHeader
              as="a"
            >
              Politics
            </ListHeader>
          </ListContent>
        </ListItem>
        <ListItem>
          <ListContent>
            <ListHeader
              as="a"
            >
              Technology
            </ListHeader>
          </ListContent>
        </ListItem>
        <ListItem>
          <ListContent>
            <ListHeader
              as="a"
            >
              Family
            </ListHeader>
          </ListContent>
        </ListItem>
        <ListItem>
          <ListContent>
            <ListHeader
              as="a"
            >
              Family
            </ListHeader>
          </ListContent>
        </ListItem>
        <ListItem>
          <ListContent>
            <ListHeader
              as="a"
            >
              Family
            </ListHeader>
          </ListContent>
        </ListItem>
        <ListItem>
          <ListContent>
            <ListHeader
              as="a"
            >
              Family
            </ListHeader>
          </ListContent>
        </ListItem>
        <ListItem>
          <Label
            as="a"
          >
            more
          </Label>
        </ListItem>
      </List>
    </GridColumn>
  </GridColumn>
  <GridColumn
    className="Advert"
  >
    <Image
      as="img"
      src="test-file-stub"
      ui={true}
    />
  </GridColumn>
  <GridColumn
    className="NewsLetter"
  >
    <header
      text="NewsLetter"
    />
    <Input
      placeholder="Search..."
      type="text"
    />
    <Button
      as="button"
      basic={true}
      color="black"
    >
      Submit
    </Button>
  </GridColumn>
</GridRow>
`;
